recipe/meta.yaml

```
@@ -43,6 +43,7 @@ test:
     - echo -105 40 | proj +proj=utm +zone=13 +ellps=WGS84
     - echo -117 30 | cs2cs +proj=latlong +datum=NAD27 +to +proj=latlong +datum=NAD83
     - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
+    - echo -92.199881 38.56694 | proj EPSG:26915 | cs2cs EPSG:26915 +to EPSG:26715
```

Based on https://github.com/conda-forge/pyproj-feedstock/pull/144, the first command below should give the wrong answer (second component at about -3.54). I run it first because I noticed in pyproj testing that running `cs2cs` changes the results of later operations (at least in pyproj).

```suggestion
    - echo 569704.57 4269024.67 | cs2cs EPSG:26915 +to EPSG:26715
    - echo -105 40 | proj +proj=utm +zone=13 +ellps=WGS84
    - echo -117 30 | cs2cs +proj=latlong +datum=NAD27 +to +proj=latlong +datum=NAD83
    - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
    - echo -92.199881 38.56694 | proj EPSG:26915 | cs2cs EPSG:26915 +to EPSG:26715
```